tests/unittests/media/server/gstplayer/genericPlayer/GstCapabilitiesTest.cpp

```diff
@@ -340,9 +359,10 @@ TEST_F(GstCapabilitiesTest, getSupportedPropertiesWithPropertiesSupported)
     EXPECT_CALL(*m_glibWrapperMock, gObjectClassListProperties(_, _))
         .WillOnce(DoAll(SetArgPointee<1>(kNumParamsSupportedByServer), Return(dummySinkParamsPtr)));
     EXPECT_CALL(*m_glibWrapperMock, gFree(dummySinkParamsPtr)).Times(1);
+    EXPECT_CALL(*m_rdkGstreamerUtilsWrapperMock, isSocAudioFadeSupported()).WillOnce(Return(true));
```

Comment:
I would keep this test as it is, and dont pass 'audio-fade' in, in this case isSocAudioFadeSupported shouldnt be called.

Then you need two separate tests, one for audio-fade property and one for soc.